internal/restconfig/restconfig.go

```diff
@@ -444,15 +438,11 @@ func (rcp *Producer) RunOnAllContexts(function PerContextFn, status reporter.Int
 
 func (rcp *Producer) overrideContextAndRun(clusterName, contextName string, function PerContextFn, status reporter.Interface) error {
 	fmt.Printf("Cluster %q\n", clusterName)
+	defer fmt.Println()
```

Comment:
Neat! I’ll have to remember that, write the end of a function backwards as `defer`s to avoid `if err` handling :wink:.